L.Map.Sync.js

```diff
@@ -21,6 +21,16 @@
             }
 
             if (!options.noInitialSync) {
+                if (options.addCircleMarker) {
```

Comment:
I would say `addCircleMarker` is not telling met what it really does. In my [syncCursor-branch](https://github.com/turban/Leaflet.Sync/compare/sync-cursor) I used `syncCursor` as the option, I would say that's more clear, you're welcome to think of a better one.